lib/rules/jest/no-vague-titles.js

```diff
@@ -0,0 +1,92 @@
+module.exports = {
+  meta: {
+    docs: {
+      description: 'Prevent the usage of vague words in test statements.',
+      category: 'Best Practices',
+      recommended: false,
+      uri:
+        'https://github.com/Shopify/eslint-plugin-shopify/blob/master/docs/rules/no-vague-titles.md',
+    },
+  },
+  create(context) {
+    const ignored = (context.options[0] && context.options[0].ignore) || [];
+
+    function isIgnoredFunctionName(node) {
+      if (node.type === 'MemberExpression') {
+        return ignored.some((method) => method === node.object.name);
+      }
+      return ignored.some((method) => method === node.callee.name);
+    }
+
+    function validate(node) {
+      if (
+        !isTestFunction(node) ||
+        isIgnoredFunctionName(node) ||
+        !containsDescription(node)
+      ) {
+        return;
+      }
+
+      const description = getDescription(node);
+
+      if (containsVagueWord(description)) {
+        context.report({
+          message: `{{ method }} statement should not contain vague words`,
```

Comment:
description instead of statement

Reply:
I think it would also be nice to briefly mention what they should do instead; write a description that more meaningfully illustrates the purpose of the test/ suite.